lib/inspec/reporters.rb

```diff
@@ -13,8 +12,6 @@ def self.render(reporter, run_data)
     name, config = reporter.dup
     config[:run_data] = run_data
     case name
-    when "cli"
-      reporter = Inspec::Reporters::CLI.new(config)
     when "json"
```

Comment:
Would part of the long term goal here see the entire case statement replaced by the line we're currently catchingn in the else? https://github.com/inspec/inspec/blob/afdcc9d2a1b76a4f0051e527015e2029095e458a/lib/inspec/reporters.rb#L31

Reply:
Yes, that's right.